lib/tunnel.js

```diff
@@ -7,6 +7,11 @@ module.exports = function setupTunnel(subdomain, port, retries = 0) {
     subdomain = require('os').userInfo().username;
   }
 
+  if (subdomain.length < 4) {
+    const pkg = require('load-pkg').sync(process.cwd());
+    subdomain = `${subdomain}-${pkg.name}`;
```

Comment:
This will a dash, but the error message I got said the the subdomain must be alphanumeric.